### PR TITLE
Footer redesign

### PR DESCRIPTION
**Pull request by harrykeefe**

### CODE REVIEW

**Comment by harrykeefe** [comment]
[(0.5) New footer design](https://trello.com/c/aF8wSpBt/5852-05-new-footer-design)

File: app/views/reimagine2/devpost/_footer.html.erb [hunk 1]
```diff
@@ -40,33 +40,56 @@
...
-          <li><a href="https://info.devpost.com/guidelines"><small>Community Guidelines</small></a></li>
-          <li><a href="https://info.devpost.com/terms"><small>Terms of service</small></a></li>
-        </ul>
-      </nav>
-    </div>
+  <div class="legal-bar-container">
+    <nav class="row">
+      <div class="columns">
+        <div class="legal-bar">
+          <div class="copyright-notice">&#169; 2021 Devpost, Inc. All rights reserved.</div>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/_footer.html.erb:83]: lets make the year dynamic and just use the current year so we don't have to do this all the time

File: app/views/reimagine2/devpost/_footer.html.erb [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-
     <div class="small-6 large-3 columns">
       <nav>
-        <h4>For Organizations</h4>
+        <h4>Portfolios</h4>
         <ul>
-          <li><a href="https://post.devpost.com/hackathons">Host an in-person hackathon</a></li>
-          <li><a href="https://post.devpost.com/online_hackathons">Host an online hackathon</a></li>
-          <li><a href="https://post.devpost.com/app_contest_resources">Hackathon best practices</a></li>
+          <li><a href="https://staging.devpost.com/my-portfolio?page=projects">Your projects</a></li>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/_footer.html.erb:31]: This needs to be dynamic for the environment. do we have domain helpers in reimagine to help us figure that out?

File: app/assets/stylesheets/reimagine2/devpost/_footer.sass [hunk 3]
```diff
@@ -1,108 +1,127 @@
...
+            text-decoration: underline
+
+      .copyright-notice
+        font-size: 0.8rem
+        margin: 0
+
+        @media #{$small}
+          font-size: 1rem
+
+      @media only screen and (min-width: 930px)
```
Comment (devpost-mzheng) [inline app/assets/stylesheets/reimagine2/devpost/_footer.sass:114]: whered 930 come from?

File: app/views/reimagine2/devpost/_footer.html.erb (hunk 2 above)
Comment (harrykeefe) [reply app/views/reimagine2/devpost/_footer.html.erb:31]: I didn't see one in the helpers, I also tried searching for "staging" to no avail. Should I make a helper?

Comment (harrykeefe) [reply app/views/reimagine2/devpost/_footer.html.erb:31]: I was planning to just update before shipping. It needs to be staging for staging so phil can test platform behavior.

File: app/views/reimagine2/devpost/_footer.html.erb (hunk 1 above)
Comment (harrykeefe) [reply app/views/reimagine2/devpost/_footer.html.erb:83]: Ok I made it dynamic (haven't pushed yet)

File: app/assets/stylesheets/reimagine2/devpost/_footer.sass (hunk 3 above)
Comment (harrykeefe) [reply app/assets/stylesheets/reimagine2/devpost/_footer.sass:114]: 930 is based on the smallest width until the text needs to wrap. Definitely need to test this with the new font as the breakpoint could be different

File: app/views/reimagine2/devpost/_footer.html.erb [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
       <nav>
-        <h4>For Organizations</h4>
+        <h4>Portfolios</h4>
         <ul>
-          <li><a href="https://post.devpost.com/hackathons">Host an in-person hackathon</a></li>
-          <li><a href="https://post.devpost.com/online_hackathons">Host an online hackathon</a></li>
-          <li><a href="https://post.devpost.com/app_contest_resources">Hackathon best practices</a></li>
+          <li><a href="<%= platform_url('/portfolio/redirect', 'page=projects') %>">Your projects</a></li>
+          <li><a href="<%= platform_url('/portfolio/redirect', 'page=hackathons') %>">Your hackathons</a></li>
+          <li><a href="https://staging.devpost.com/settings">Settings</a></li>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/_footer.html.erb:33]: you still need to change this last one

File: app/helpers/reimagine2/url_helper.rb [hunk 5]
```diff
@@ -20,5 +20,26 @@ def reimagine_post_url(opts = {})
     def post_subdomain?
       request.subdomains.first == "post"
     end
+      
+    def platform_url(path, query)
+      return URI::HTTPS.build(host: current_env_host, path: path) if query.empty?
```
Comment (devpost-mzheng) [inline app/helpers/reimagine2/url_helper.rb:25]: can u make this DRY by merging the arguments with https://apidock.com/ruby/Hash/merge. Also consider default arguments

Comment (harrykeefe) [reply app/helpers/reimagine2/url_helper.rb:25]: yeah I can that'd be much better

**Pull request merged**